### PR TITLE
Feature: parking and stopping forbidden signs on "wegvak" details pane

### PR DESCRIPTION
**Pull request by RubenSibon**
Display "parkeerverbod" (E01) or "verbod om te stoppen" (E02) sign in details pane of a "wegvak" feature.

### CODE REVIEW

File: src/pages/ProhibitorySigns/components/ScenarioWizard/ScenarioWizard.test.tsx
Comment (TerryvanWalen) [inline src/pages/ProhibitorySigns/components/ScenarioWizard/ScenarioWizard.test.tsx:9]: Why do we need to wait 10 seconds? (comment ook voor andere plekken waar dit gebeurd)

File: src/pages/LoadUnload/components/DetailFeature/RoadSectionLoadUnload.tsx
Comment (TerryvanWalen) [inline src/pages/LoadUnload/components/DetailFeature/RoadSectionLoadUnload.tsx:79]: Do you want to be this specific or should we check if cur.additional_info is in SignCodeToImg?

File: .env.development
Comment (TerryvanWalen) [inline .env.development:1]: You probably thought of this already but can we automate this in any way? Adding another env and/or adding a run script in package.json for running locally?

**Review by TerryvanWalen — COMMENTED** [review]
Looking very good. I only looked at the code, haven't seen it live but I think you were planning to do that with Manilde? I love the functional code!

Not sure if I should approve or if somebody else should look over it?

File: src/pages/ProhibitorySigns/components/ScenarioWizard/ScenarioWizard.test.tsx
Comment (RubenSibon) [reply src/pages/ProhibitorySigns/components/ScenarioWizard/ScenarioWizard.test.tsx:9]: Many tests often failed within the default 5 second timeout. By doubling that amount for affected tests no more timeouts occur (at least on my machine).

This is not really a robust solution, but there are many issues with these tests and they should be properly looked at, at some point.

File: src/pages/LoadUnload/components/DetailFeature/RoadSectionLoadUnload.tsx
Comment (RubenSibon) [reply src/pages/LoadUnload/components/DetailFeature/RoadSectionLoadUnload.tsx:79]: No, because as per request we do not want to show all the possible traffic signs that may exist in "additional info". For now, we are specifically interested in E01 and E02.

File: .env.development
Comment (RubenSibon) [reply .env.development:1]: Yeah I did. I'll see if I can make a solution were the developer has a simple `.env` that overrides whatever is in `.env.development`.

File: src/pages/ProhibitorySigns/components/ScenarioWizard/ScenarioWizard.test.tsx
Comment (RubenSibon) [reply src/pages/ProhibitorySigns/components/ScenarioWizard/ScenarioWizard.test.tsx:9]: So, I'm not going to fix the tests properly in this PR, but I'll add an issue to the backlog.

File: .env.development
Comment (RubenSibon) [reply .env.development]: There was actually already a solution: use `.env.development.local`.

**Comment by RubenSibon** [comment]
@TerryvanWalen, I think I have resolved all your remarks. Can you complete the review and approve if it's ready for merge?

**Review by manilde — APPROVED** [review]

**Pull request merged**
